front/src/common/Loader.tsx

```diff
@@ -19,7 +19,7 @@ const LoaderSNCF: FC<{
 }> = ({ msg = '', position = 'center' }) => (
   <div className={`loader ${position}`}>
     <Spinner />
-    <div className="text-center mt-2">{msg}</div>
+    <div className="scenario-loader-msg">{msg}</div>
```

Comment:
This component in in common , you can't pass it contextual class name like this

Reply:
The new WaitingMessage component should be in common too maybe ?

Reply:
Maybe but not in this form, if you want it to be a common component, you'll have to remove `<ScenarioLoader/>` from it and try to use more props

Reply:
Done :)